mala/network/trainer.py

```diff
@@ -412,15 +486,27 @@ def __prepare_to_train(self, optimizer_dict):
 
         # Choose an optimizer to use.
         if self.parameters.trainingtype == "SGD":
+            printout("Using torch.optim.SGD optimizer...", min_verbosity=2)
             self.optimizer = optim.SGD(self.network.parameters(),
-                                       lr=self.parameters.learning_rate,
-                                       weight_decay=self.parameters.
-                                       weight_decay)
+                                      lr=self.parameters.learning_rate,
+                                      weight_decay=self.parameters.
+                                      weight_decay)
         elif self.parameters.trainingtype == "Adam":
+            printout("Using torch.optim.Adam optimizer...", min_verbosity=2)
             self.optimizer = optim.Adam(self.network.parameters(),
                                         lr=self.parameters.learning_rate,
                                         weight_decay=self.parameters.
                                         weight_decay)
+        elif self.parameters.trainingtype == "FusedAdam":
+            printout("Using Apex FusedAdam optimizer...", min_verbosity=2)
```

Comment:
I see that instead of the `FusedAdam` from the `apex` library, you've opted to use the new `fused` option in the native `torch.optim.Adam` optimizer, which is a good change. The printout here should be updated to reflect that though.

Reply:
Good point! On second thought, I removed all the printouts - they shouldn't be necessary, since choosing the optimizer is done via the parameters and therefore accesible via there.